auto_nag/config.py

```diff
@@ -15,19 +15,25 @@ class MyConfig(config.Config):
     def __init__(self):
         super(MyConfig, self).__init__()
         if not os.path.exists(MyConfig.PATH):
-            self.conf = {"bz_api_key": ""}
+            self.conf = {"bz_api_key": "", "bz_api_key_nomail": ""}
         else:
             with open(MyConfig.PATH) as In:
                 self.conf = json.load(In)
-                if not self.conf.get("bz_api_key", None):
-                    raise Exception(
-                        "Your config.json file must contain a Bugzilla token"
-                    )
+
+        if "bz_api_key" not in self.conf:
+            raise Exception("Your config.json file must contain a Bugzilla token")
+
+        if "bz_api_key_nomail" not in self.conf:
+            raise Exception(
+                "Your config.json file must contain a Bugzilla token for an account that doesn't trigger bugmail (for testing, you can use the same token as bz_api_key)"
+            )
```

Comment:
Just had an idea, only worth it if it's trivial to implement: we could raise an exception only when not in dry run mode

Reply:
This will be invoked when loading the `autonag` module (before running the tool). Thus, it will not be aware of the dry-run.